README.md

```diff
@@ -22,6 +22,11 @@ This project is provided under the [Eclipse Public License 2.0](LICENSE)
 ### Set up the project in eclipse
 Import repository clone root folder as "Existing Projects into Workspace" and select "Search for nested projects" before clicking "Finish".
 
+#### Lombok
+We use Lombok to generate boilerplate. Lombok is compatible with various IDEs, find the setup instructions here [Lombok Eclipse Setup](https://projectlombok.org/setup/eclipse).
```

Comment:
The word "code" is missing, isn't it?

Reply:
If you wish I can add 'code' but from what I know, boilerplate and boilerplate code are used interchangeably.